packages/core/src/payment/strategies/ppsdk/step-handler/step-handler.ts

```diff
@@ -11,15 +12,15 @@ export class StepHandler {
         private  _continueHandler: ContinueHandler
     ) {}
 
-    handle(response: PaymentsAPIResponse): Promise<void> {
+    handle(response: PaymentsAPIResponse, callback?: (additionalAction: PaymentAdditionalAction) => Promise<void>): Promise<void> {
         const { body } = response;
 
         if (isSuccess(body)) {
             return handleSuccess();
         }
 
         if (isContinue(body)) {
-            return this._continueHandler.handle(body);
+            return this._continueHandler.handle(body, callback);
```

Comment:
`callback` needs to be triggered for all type of responses, not just `continue`. Otherwise, we should specify that this is a callback for a specific step, i.e.: `continueCallback`.